backend/auth.js

```diff
@@ -21,7 +21,7 @@ exports.register = function (
         if (scheme !== 'gi') return reply(Boom.badRequest('Bad authentication'))
 
         try {
-          json = JSON.parse(b642str(json))
+          json = JSON.parse(b64ToStr(json))
```

Comment:
this is definitely a way [more fortunate naming convention](https://www.barnesandnoble.com/w/clean-code-robert-c-martin/1101628669?ean=9780132350884&pcta=n&st=PLA&sid=BNB_DRS_New+Core+Shopping+Textbooks_00000000&2sid=Google_&sourceId=PLGoP211688&gclid=CjwKCAjw14rbBRB3EiwAKeoG_2qylKsEFBjK18huqfcAmFHBuuIgWLCxW9-qy_JjC0JcRXd0LhXz9BoCKHwQAvD_BwE) for this function.

Reply:
... that's a book. Can you be more specific?

Reply:
Oh, I misread your comment, thanks!